content/administration/maintain/domain_names.rst

```diff
@@ -0,0 +1,324 @@
+
+.. tabs::
+
+   .. group-tab:: Odoo Online
+
+      Open your `database manager <https://www.odoo.com/my/databases>`_, click on the *settings*
+      button next to your database, on *Domain names*, and then on *Use my own domain* at the bottom
+      of the right column.
+
+      Type the domain name you want to add to this database, then click on *Verify* to check if the
+      the CNAME record is correctly configured. Once done, click on *I confirm, it's done*.
+
+      .. image:: domain_names/online-map.png
+         :align: center
+         :alt: Verification of the CNAME records of a domain name before mapping it with a database
+
+   .. group-tab:: Odoo.sh
+
+      Go to :menuselection:`Branches --> select your branch --> Settings --> Custom domains`, type
+      the domain name you want to add to this database, then click on *Add domain*.
+
+      .. image:: domain_names/odoo-sh-map.png
+         :align: center
+         :alt: Mapping a domain name with an Odoo.sh branch
+
+      .. seealso::
+         - :ref:`Odoo.sh branches: settings tab <odoosh-gettingstarted-branches-tabs-settings>`
+
+.. warning::
+   Make sure to :ref:`add a CNAME record <domain-name/cname>` to your domain name's DNS **before**
+   mapping your domain name with your Odoo database.
+
+   Failing to do so may impede the validation of the :ref:`SSL certificate <domain-name/ssl>` and
+   would result in a *certificate name mismatch* error. This is often displayed by web browsers as a
+   warning such as *"Your connection is not private"*.
+
+   If this is the case and you have added the domain name to your database's settings less than five
+   days ago, wait 24 hours as the validation may still happen. Otherwise, please `submit a support
+   ticket <https://www.odoo.com/help>`_ including screenshots of your CNAME records.
+
+.. _domain-name/ssl:
+
+SSL encryption (HTTPS protocol)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**SSL encryption** is an encryption-based Internet security protocol. It allows your visitors to
+navigate your website through a secure connection, which appears as an ``https://`` protocol at the
+beginning of your web address, rather than a non-secure ``http://`` protocol.
+
+Odoo generates a separate SSL certificate for each domain :ref:`mapped in the database manager
+<domain-name/db-map>`, using integration with `Let's Encrypt Certificate Authority and ACME protocol
+<https://letsencrypt.org/how-it-works/>`_.
+
+.. note::
+   - The certificate generation may take up to 24h.
+   - Several attempts to validate your certificate are made during the five days following the
+     moment you add your domain name in your database's settings.
+   - If you already use another service, you can keep using it or simply change for Odoo.
+
+.. _domain-name/web-base-url:
+
+Web base URL of a database
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+   If you have Odoo Website, you can disregard this part and directly :ref:`map your domain name
```

Comment:
As of v15 [#](https://github.com/odoo/odoo/pull/68201)

Reply:
Thanks for this!
I removed the following points from this PR and will add them from v15:

- this note
- "It defines your domain name as the base URL for your database, including the various portal links sent to customers." in **Map your domain name with your website**
- ```
  .. note::
     If you have multiple websites and companies on your database, make sure that you select the
     right **Company** in the website settings, next to the **Domain** settings. This indicates which
     URL to use as base URL for the links sent to your customers.
  ```